xray/src/build_quadtree.rs

```diff
@@ -125,7 +125,7 @@ pub fn run<T: Extension>(data_provider_factory: DataProviderFactory) {
     attempt_increasing_rlimit_to_max();
 
     let args = parse_arguments::<T>();
-    let resolution = args
+    let pixel_resolution = args
         .value_of("resolution")
         .unwrap()
         .parse::<f64>()
```

Comment:
nit: resolution -> pixel_resolition

Reply:
I didn't want to change the current interface... Could do that in a separate PR, if required.

Reply:
Nevermind, I misread the code.